symfony/plugins/orangehrmTimePlugin/Api/Model/TimesheetActionLogModel.php

```diff
@@ -19,43 +19,55 @@
 
 namespace OrangeHRM\Time\Api\Model;
 
+use DoctrineProxies\__CG__\OrangeHRM\Entity\EmployeeTerminationRecord;
 use OrangeHRM\Core\Api\V2\Serializer\ModelTrait;
 use OrangeHRM\Core\Api\V2\Serializer\Normalizable;
 use OrangeHRM\Entity\TimesheetActionLog;
+use OrangeHRM\Entity\User;
 
 class TimesheetActionLogModel implements Normalizable
```

Comment:
Do we need to change this model after you make the `performedUser` nullable in the entity class?

Reply:
User Entity does not have a direct connection with Timesheet so changing the TimesheetActionLog Model and Entity should be enough right?

Reply:
Set the model back to the original. Made a small change in blade instead to handle the null value in performedBy